piezo_web_app/PiezoWebApp/src/services/spark_job/spark_job_service.py

```diff
@@ -39,10 +41,10 @@ def delete_job(self, job_name):
                 job_name,
                 body
             )
-
-            msg = f'"{job_name}" deleted' if api_response['status'] == "Success"\
+            msg = f'"{job_name}" deleted' if api_response['status'] == "Success" \
                 else f'Trying to delete job "{job_name}" resulted in status: {api_response["status"]}'
             self._logger.debug(msg)
+            self._delete_spark_ui_components(job_name, body)
```

Comment:
Returning a message to the user has now been implemented. However, it is likely that users won't delete their jobs themselves that often and it will be left to the tidy jobs handler to keep the cluster tidy. Here any message is redundant.

Reply:
Also issues arise if a component fails to initiate then the user will receive an error when trying to delete it as it won't exist.

Reply:
0d30a4e

Reply:
db9feac